src/chart/rendering/drawlegend.h

```diff
@@ -25,20 +25,35 @@ class DrawLegend : public DrawingContext
 	    *rootStyle.plot.marker.colorGradient};
 
 private:
+	struct ColorGradientSetter
+	{
+		Geom::Line line;
+		Gfx::ColorGradient gradient;
```

Comment:
Would be good to have a `Gfx::LinearGradient` type for line+gradient

Reply:
but for this class, the "line" and the colors are changing always.